modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -935,4 +938,6 @@ Curl::
 * xref:reference:node-configuration-sample.adoc[] (search for `schema_registry`)
 * xref:manage:monitoring.adoc#service-level-queries[Monitor Schema Registry service-level metrics]
 * xref:deploy:deployment-option/self-hosted/manual/node-property-configuration.adoc[Configure broker properties for Schema Registry]
```

Comment:
We should conditionalize these links. Cloud docs should have links to node property configs or node-configuration-sample.adoc for example.